transports/transports.go

```diff
@@ -2,52 +2,24 @@ package transports
 
 import (
 	"fmt"
-	"strings"
 
-	"github.com/containers/image/directory"
-	"github.com/containers/image/docker"
-	"github.com/containers/image/docker/daemon"
-	ociLayout "github.com/containers/image/oci/layout"
-	"github.com/containers/image/openshift"
-	"github.com/containers/image/storage"
 	"github.com/containers/image/types"
-	"github.com/pkg/errors"
 )
 
 // KnownTransports is a registry of known ImageTransport instances.
 var KnownTransports map[string]types.ImageTransport
 
 func init() {
 	KnownTransports = make(map[string]types.ImageTransport)
-	// NOTE: Make sure docs/policy.json.md is updated when adding or updating
-	// a transport.
```

Comment:
Please preserve this comment (in the import block of `alltransports.go` probably).